package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "powershell",
-  "version": "1.0.0",
+  "version": "2.0.0",
```

Comment:
Does upgrading to node20 require a major version jump?

Reply:
Yes, I'll release a new major version 2.0.0.
We'ver done this for Login Action.